online-banking-cz/src/main/java/com/adyen/checkout/onlinebankingcz/OnlineBankingCZSpinnerView.kt

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by atef on 8/9/2022.
+ */
+
+package com.adyen.checkout.onlinebankingcz
+
+import android.content.Context
+import android.net.Uri
+import android.text.method.LinkMovementMethod
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.AdapterView
+import androidx.browser.customtabs.CustomTabsIntent
+import androidx.lifecycle.LifecycleOwner
+import com.adyen.checkout.components.PaymentComponentState
+import com.adyen.checkout.components.model.payments.request.OnlineBankingCZPaymentMethod
+import com.adyen.checkout.components.ui.adapter.SimpleTextListAdapter
+import com.adyen.checkout.components.ui.util.ThemeUtil
+import com.adyen.checkout.components.ui.view.AdyenLinearLayout
+import com.adyen.checkout.core.log.LogUtil
+import com.adyen.checkout.core.log.Logger
+import com.adyen.checkout.onlinebankingcz.databinding.OnlineBankingCzSpinnerLayoutBinding
+
+class OnlineBankingCZSpinnerView @JvmOverloads constructor(
```

Comment:
I would leave out `Spinner` of this name. The issuer list "spinner" view is already a weird name, but at least it's got a spinner, this view doesn't have a spinner